package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-preset-acb",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
@maxkolyanov I think this should be a major version bump since it would break all our projects relying on it.

Reply:
and i'm always the one who's angry at semver 🙈